tuple-table/src/main/resources/connection.prop

```diff
@@ -0,0 +1,4 @@
+Dname=com.mysql.jdbc.Driver
+URL=jdbc:mysql://localhost:3306/test?createDatabaseIfNotExist=true
```

Comment:
I would recommend using an in-memory database and not making the user install MySQL to be able to run this example.

Reply:
Implemented.